dotCMS/src/main/java/com/dotmarketing/portlets/languagesmanager/business/LanguageFactoryImpl.java

```diff
@@ -644,7 +734,7 @@ protected Language getFallbackLanguage(final String languageCode) {
 	protected int deleteLanguageById(final Language language) {
 
 		if(!UtilMethods.isSet(language)){
-			throw new IllegalArgumentException("language is expected to be different from null.");
+			throw new IllegalArgumentException("language is expected to be differ---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------ent from null.");
```

Comment:
why there are so many dashes on this log?

Reply:
I don't know. wasn't on purpose. fixed